doc/man3/X509_LOOKUP.pod

```diff
@@ -195,21 +196,30 @@ or NULL on error.
 X509_LOOKUP_init() and X509_LOOKUP_shutdown() return 1 on success, or
 0 on error.
 
-X509_LOOKUP_ctrl() returns -1 if the B<X509_LOOKUP> doesn't have an
+X509_LOOKUP_ctrl_ex() and X509_LOOKUP_ctrl()
+return -1 if the B<X509_LOOKUP> doesn't have an
 associated B<X509_LOOKUP_METHOD>, or 1 if the X<509_LOOKUP_METHOD>
 doesn't have a control function.
 Otherwise, it returns what the control function in the
-B<X509_LOOKUP_METHOD> returns, which is usually 1 on success and 0 in
-error.
+B<X509_LOOKUP_METHOD> returns, which is usually 1 on success and 0 on error
+but could also be -1 on failure.
 
 X509_LOOKUP_get_store() returns a B<X509_STORE> pointer if there is
 one, otherwise NULL.
 
-X509_LOOKUP_by_subject_ex(), X509_LOOKUP_by_subject(),
+X509_LOOKUP_by_subject_ex()
+returns -1 if there is no B<X509_LOOKUP_METHOD> or that
```

Comment:
This is an API break and I do not think we want it. In general changing return values of existing functions especially if they are documented is not a good idea.

Reply:
> This is an API break 

A pity we did not finish this during the preparation of 3.0, where `X509_LOOKUP_by_subject_ex()` was freshly added.
See above https://github.com/openssl/openssl/pull/14417#issuecomment-817169538.
I had intended this PR as a fixup that time, but then this got marked **Post 3.0.0**.

To avoid the API issue that meanwhile came up, I've retracted the change of the return value.